test_conformance/spirv_new/test_extinst_printf.cpp

```diff
@@ -232,8 +234,9 @@ lX = 4
         return TEST_SKIPPED_ITSELF;
     }
 
-    return printf_operands_helper(device, "printf_operands_scalar_int64",
-                                  "printf_operands_scalar_int64", expected, 4L);
+    return printf_operands_helper<int64_t>(
+        device, "printf_operands_scalar_int64", "printf_operands_scalar_int64",
+        expected, 4L);
```

Comment:
Ah, good catch.

I guess we could also do something like this?  I don't know if this makes it any more obvious what's supposed to be happening:

```suggestion
    return printf_operands_helper(device, "printf_operands_scalar_int64",
                                  "printf_operands_scalar_int64", expected,
                                  static_cast<cl_long>(4));
```

Reply:
That may also work, but to me it feels a little less obvious and more fragile than just making the template type explicit.

Reply:
Thanks, waiting until the call tomorrow is fine!